tests/src/cgeo/geocaching/files/ProgressInputStreamTest.java

```diff
@@ -9,7 +9,7 @@
 public class ProgressInputStreamTest extends TestCase {
 
     public static void testRead() throws Exception {
-        ProgressInputStream stream = new ProgressInputStream(IOUtils.toInputStream("test"));
+        ProgressInputStream stream = new ProgressInputStream(IOUtils.toInputStream("test", (String) null));
```

Comment:
Short remark: The original method already used default encoding:So 

> Converts the specified CharSequence to an input stream, encoded as bytes using the default character encoding of the platform.

So the PR did not make it worse. However, I will update the PR to use UTF-8.